sdk/web-pubsub/web-pubsub-client/review/web-pubsub-client.api.md

```diff
@@ -310,7 +310,7 @@ export interface WebPubSubMessageBase {
 
 // @public
 export interface WebPubSubResult {
-    ackId: number;
+    ackId?: number;
     isDuplicated: boolean;
```

Comment:
If this is a one way fire and forget, does this mean `isDuplicated` will always be false as well?